ci-operator/step-registry/upi/install/aws/cluster-custom/upi-install-aws-cluster-custom-ref.yaml

```diff
@@ -0,0 +1,26 @@
+ref:
+  as: upi-install-aws-cluster-custom
+  from: upi-installer
+  grace_period: 10m
+  commands: upi-install-aws-cluster-custom-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+  - name: BOOTSTRAP_INSTANCE_TYPE
+    default: "i3.large"
+  - name: MASTER_INSTANCE_TYPE
+    default: "m5.xlarge"
+  - name: WORKER_INSTANCE_TYPE
+    default: "m5.xlarge"
+  - name: ADD_INGRESS_RECORDS_MANUALLY
+    default: "no"
+  dependencies:
+  - name: "release:latest"
+    env: OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE
+  documentation: >-
+    This step deploys a UPI cluster to the CI AWS project.
+    It allows you to create app dns records manually via setting ADD_INGRESS_RECORDS_MANUALLY
+    And supoprts private cluster and disconnected & private cluster.
+    The VPC and basion host are created by aws-provision-vpc* and aws-provision-bastionhost
```

Comment:
there's a lot of overlap, and also a lot of divergence, between this new step and the existing `upi-install-aws-cluster`.  I'm not a maintainer of either, so :shrug: y'all can do whatever makes most sense to you, but it seems like the two maintainer sets may end up doing duplicate work, and anyone looking for an AWS UPI install step to consume will have a hard time selecting which is the best fit for their goals.

Reply:
Yes, it's true that there are a lot of overlaps and differences.
For our tests, we need to cover many test configurations, so we did some customization, but we don't want to break up the existing processes.

Reply:
so docs like "use this step if you need more flexibility than upi-install-aws-cluster provides"?